main/main.cpp

```
@@ -9,48 +9,68 @@
+
+// Set measurement interval and callback, then start continuous measurements
+sensor.setMeasurementInterval(MEASUREMENT_INTERVAL_MS); // 5 second interval
+sensor.setMeasurementCallback(measurementHandler);
+if (!sensor.startContinuousMeasurement()) {
+    printf("Failed to start continuous measurement task (possibly insufficient memory)\n");
+    return;
+}
+
+// Main task can exit or delay indefinitely
+vTaskDelay(portMAX_DELAY);
     return;
```

Clang (14.0.6)

[warning] redundant return statement at the end of a function with a void return type

(readability-redundant-control-flow)